server/models/Notifications.js

```diff
@@ -0,0 +1,26 @@
+const mongoose = require("mongoose");
+
+const notificationSchema = new mongoose.Schema({
+    type: {
+        type: String,
+        default: ""
+    },
+    title: {
+        type: String,
+        default: ""
+    },
+    description: {
+        type: String,
+        default: ""
+    },
+    read: {
+        type: Boolean,
+        default: false
+    },
+    date: {
```

Comment:
Optional: You can use mongoose timestamps here - it will automatically manage fields like 'createdAt' and 'updatedAt'